content/blog/infrastructure-testing-got-better/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Infrastructure Testing Just Got Better with Pulumi"
+date: 2020-05-13
+meta_desc: "Pulumi accelerates infrastructure testing 6000x with mocking and unit tests."
```

Comment:
The 6000x is unqualified. Also, later I saw we came to this 6000x from 6000% in which case, this ought to be 60x. Maybe let's go with "60x faster"?

Reply:
agreed